addons/portal/static/src/signature_form/signature_form.xml

```diff
@@ -11,7 +11,7 @@
                     <t t-else="">Click here to see your document.</t>
                 </a>
             </div>
-            <t t-else="">
+            <t t-if="!isSignatureHidden">
```

Comment:
This is not a stable diff: you are *removing* an attribute (t-else), which custo might rely on. Plus should not that be a t-elif? Instead, in stable, maybe you could just make this area not editable? (o_not_editable class). And/or hide it with the `css_editable_mode_hidden` class?